backend/src/bundles/auth/auth.controller.ts

```diff
@@ -76,6 +93,51 @@ class AuthController extends ControllerBase {
             payload: await this.authService.signUp(options.body),
         };
     }
+    /**
+     * @swagger
+     * /auth/sign-in:
+     *    post:
```

Comment:
Did you check swagger api for endpoint ?

Reply:
Yeah request are working in swagger ui